backend/small_eod/user/models.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib.sessions.models import Session
+
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+from guardian.mixins import GuardianUserMixin
+
+
+class User(GuardianUserMixin, AbstractUser):
+    def __str__(self):
+        return self.username
```

Comment:
Czy jest to konieczne? Nie wystarczy ```pass```?

Reply:
Wzorowalem sie na User modelu z ktoregos projektu watchdog. Zmieniam na pass i usuwam GuardianUserMixin